requirements.txt

```diff
@@ -0,0 +1 @@
+cerberus==1.3.4
```

Comment:
Almost with you could GitHub post to slack when a specific file is changed.

Reply:
Couldn't find anything to link to Slack directly, but this could be used to email individuals: https://app.github-file-watcher.com/